modyn/supervisor/supervisor.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=no-name-in-module
```

Comment:
We should add this to the individual imports in the past case, if possible, instead of globally disabling this check

Reply:
done.